sklearn/preprocessing/_data.py

```diff
@@ -1151,7 +1151,7 @@ class RobustScaler(TransformerMixin, BaseEstimator):
     The IQR is the range between the 1st quartile (25th quantile)
     and the 3rd quartile (75th quantile).
 
-    Centering and scaling happen independently on each feature by
+    Centering and scaling happens independently on each feature by
```

Comment:
there are two things happening, hence "happen", revert please.

Reply:
Thank you, for that working on it